legal-api/src/legal_api/reports/report.py

```diff
@@ -38,14 +38,16 @@
 )
 from legal_api.models.business import ASSOCIATION_TYPE_DESC
 from legal_api.reports.registrar_meta import RegistrarInfo
-from legal_api.services import MinioService, VersionedBusinessDetailsService
+from legal_api.services import Flags, MinioService, VersionedBusinessDetailsService
```

Comment:
Did you try this? if you are getting error then move the import to _get_template_data function
```suggestion
from legal_api.services import flags, MinioService, VersionedBusinessDetailsService
```

Reply:
Yes this works, just had to move the import order due to this:

![image](https://github.com/user-attachments/assets/58de4433-990e-4fe2-a639-38fe226b9b2d)